compiler/src/iree/compiler/Dialect/Flow/Transforms/test/form_dispatch_regions.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --pass-pipeline="builtin.module(util.func(iree-flow-form-dispatch-regions))" --split-input-file %s | FileCheck %s
+// RUN: iree-opt --pass-pipeline="builtin.module(util.func(iree-flow-form-dispatch-regions{aggressive-fusion=true}))" --split-input-file %s | FileCheck %s
```

Comment:
Are we going to miss the tests for default path? Do we add a new mlir file to test `aggressive-fusion=true`? I don't suggest to add an additional `FileCheck` in this file because some of tests seem redundant to the aggressive fusion.